charts/capsule-proxy/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for capsule-proxy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: quay.io/clastix/capsule-proxy
+  pullPolicy: IfNotPresent
+  tag: ""
+options:
+  logLevel: '5'
+  k8sControlPlaneUrl: https://kubernetes.default.svc
+  capsuleUserGroup: capsule.clastix.io
+  oidcUsernameClaim: referred_username
```

Comment:
it should be `preferred_username` instead of `referred_username`

Reply:
fixed